src/devices/Mhz19b/README.md

```diff
@@ -0,0 +1,23 @@
+﻿# MH-Z19B CO2-Sensor
```

Comment:
Please also add a 'category.txt' file at the root of the project. see other sensors for the example. It will place the sensor in the right category .And I guess 'Gas sensors'

Reply:
I'll add.

Reply:
Done